FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -1171,6 +1172,7 @@ extension ComposeViewController {
         guard recipient.email.isValidEmail else {
             updateRecipient(
                 email: recipient.email,
+                name: recipient.name,
```

Comment:
Instead of 

```swift
        updateRecipient(
            email: recipient.email,
            name: recipient.name,
            state: state,
            keyState: recipient.keyState
        )
```

we could maybe

```swift
        updateRecipient(
            recipient: recipient,
            state: state
        )
```

Reply:
Aha, I see.
There're another calls to that function which don't have recipient infos and I think we may leave this code as it is now.

![image](https://user-images.githubusercontent.com/77344191/161942168-31d7f494-0610-47e3-9978-cd435a03383b.png)

Reply:
Sure. Add this info to the issue I filed, for future context.

Reply:
Or maybe I didn't file any issue? Can close for now.

Reply:
I think you didn't file new issue.